tsconfig.json

```diff
@@ -1,4 +1,9 @@
 {
   "extends": "@cremalab/scripts-web/configs/tsconfig.json",
-  "include": ["src"]
-}
\ No newline at end of file
+  "include": [
+    "src"
+  ],
+  "compilerOptions": {
+    "noFallthroughCasesInSwitch": true
+  }
```

Comment:
This gets automatically added whenever you run the app, It looks like we'll need to add this setting to @cremalab/scripts-web/configs/tsconfig.json